packages/paste-core/components/chat-composer/stories/container.stories.tsx

```diff
@@ -0,0 +1,283 @@
+import { Button } from "@twilio-paste/button";
+import { Checkbox } from "@twilio-paste/checkbox";
+import { CustomizationProvider } from "@twilio-paste/customization";
+import { AttachIcon } from "@twilio-paste/icons/esm/AttachIcon";
+import { DownloadIcon } from "@twilio-paste/icons/esm/DownloadIcon";
+import { SendIcon } from "@twilio-paste/icons/esm/SendIcon";
+import { useTheme } from "@twilio-paste/theme";
+import * as React from "react";
+
+import {
+  ChatComposer,
+  ChatComposerActionGroup,
+  ChatComposerAttachment,
+  ChatComposerAttachmentCard,
+  ChatComposerAttachmentDescription,
+  ChatComposerAttachmentGroup,
+  ChatComposerAttachmentLink,
+  ChatComposerContainer,
+} from "../src";
+import type { ChatComposerProps } from "../src";
+
+export default {
+  title: "Components/Chat Composer/Container",
+  component: ChatComposer,
+};
+
+const defaultConfig: ChatComposerProps["config"] = {
+  namespace: "foo",
+  onError: (error: Error) => {
+    throw error;
+  },
+};
+
+export const Default: StoryFn = () => {
+  return (
+    <ChatComposerContainer>
+      <ChatComposer config={defaultConfig} ariaLabel="Basic chat composer" placeholder="Type here..." />
+      <ChatComposerActionGroup>
+        <Button variant="secondary_icon" size="reset">
+          <AttachIcon decorative={false} title="Attach" />
+        </Button>
+        <Button variant="primary_icon" size="reset">
+          <SendIcon decorative={false} title="Send" />
+        </Button>
+      </ChatComposerActionGroup>
+    </ChatComposerContainer>
+  );
+};
+
+Default.storyName = "Default";
+
+export const ContainedVariant: StoryFn = () => {
+  return (
+    <ChatComposerContainer variant="contained">
+      <ChatComposer config={defaultConfig} ariaLabel="Basic chat composer" placeholder="Type here..." />
+      <ChatComposerActionGroup>
+        <Button variant="secondary_icon" size="reset">
+          <AttachIcon decorative={false} title="Attach" />
+        </Button>
+        <Button variant="primary_icon" size="reset">
+          <SendIcon decorative={false} title="Send" />
+        </Button>
+      </ChatComposerActionGroup>
+    </ChatComposerContainer>
+  );
+};
+
+ContainedVariant.storyName = "Contained Variant";
+
+export const ContainedVariantWithAttachments: StoryFn = () => {
+  return (
+    <ChatComposerContainer variant="contained">
+      <ChatComposer config={defaultConfig} ariaLabel="Basic chat composer" placeholder="Type here..." />
+      <ChatComposerActionGroup>
+        <Button variant="secondary_icon" size="reset">
+          <AttachIcon decorative={false} title="Attach" />
+        </Button>
+        <Button variant="primary_icon" size="reset">
+          <SendIcon decorative={false} title="Send" />
+        </Button>
+      </ChatComposerActionGroup>
+      <ChatComposerAttachmentGroup>
+        <ChatComposerAttachmentCard onDismiss={() => {}}>
+          <ChatComposerAttachment attachmentIcon={<DownloadIcon decorative />}>
```

Comment:
praise: really nicely aligned API between composer attachment and chat log attachment